### DISCUSSION TITLE
SNZB-02P  not suported in domoticz

### DISCUSSION THREAD

**Discussion by florinssk** [Koenkk/zigbee2mqtt#21133] [Devices]
I use domoticz

Version: 2024.1
Build Hash: 5279c8261
Compile Date: 2024-01-01 13:59:22
dzVents Version: 3.1.8

Zigbee2MQTT
sonoff dongle P

I'm trying to add 2 SNZB-02P sensors and I can't. do you have any idea if they are compatible or is it another problem for me?
It appears on the Zigbee2MQTT page but with Unsupported but I don't see them in domoticz in the device.

![domoticz](https://github.com/Koenkk/zigbee2mqtt/assets/76791568/96a74487-4c3e-4340-b21f-0b9f82b2f1c3)

**Comment by florinssk** [comment]
any solution or ideea ?

thx

**Reply by ralphkillhertz** [reply]
"I have exactly the same problem and can't find any information about it. I've seen other people pair them without any issues in videos, could it have something to do with the latest update of zigbee2mqtt? Does anyone know anything?"

**Reply by ralphkillhertz** [reply]
I can't get their SONOFF SNZB-02P sensors to pair with Zigbee2mqtt. Every other device I have works just fine – they're detected right away, but these sensors seem invisible when trying to pair them. The funny thing is, my system does have a SONOFF Zigbee 3.0 P gateway, so if anyone could shed some light on this issue, I'd really appreciate it. Thanks in advance."

**Reply by SylvainGelly** [reply]
I use Home Assistant with Zigbee2mqtt and I have the same issue in that the devices are marked unsupported.
Please note that the OP actually solved their issue in https://www.domoticz.com/forum/viewtopic.php?t=41755: "after update plugin: v.3.2.0-beta, zigbee2mqtt: v.1.29.2 to plugin: v.3.2.0-beta, zigbee2mqtt: v.1.35.2 it's working."

I have the latest stable version installed (v1.36.1-1) and yet I do have the issue (or at least a similar one), so it might not be so simple. If anyone has ideas or solutions please share.
